homeassistant/components/samsungtv/config_flow.py

```diff
@@ -78,15 +77,77 @@ def _get_entry(self):
             data=data,
         )
 
+    async def _async_set_device_unique_id(self, raise_on_progress=True):
+        """Set device unique_id."""
+
+        if self._id:
+            await self.async_set_unique_id(
+                self._id, raise_on_progress=raise_on_progress
+            )
+            self._abort_if_unique_id_configured()
+
+        if not self._device_info:
+            await self._async_get_and_check_device_info()
+
+        if uuid := self._device_info.get("device", {}).get(ATTR_UPNP_UDN.lower()):
+            self._id = uuid
+
+        if not self._id:
```

Comment:
Since we are sending them though here on `async_step_user` this will break adding TVs that are using the legacy bridge (SamsungTVLegacyBridge) since they won't have a unique id.